bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,942 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="souliss"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+
+	<bridge-type id="gateway">
+		<label>Gateway</label>
+		<description>Represents a Souliss Gateway.</description>
+
+		<config-description>
+			<parameter name="GATEWAY_IP_ADDRESS" type="text" required="true">
+				<label>IP or Host Name</label>
+				<description>Will be resolved by discovery if auto configured
+				</description>
+			</parameter>
+			<parameter name="GATEWAY_PORT_NUMBER" type="integer" required="true">
+				<label>Gateway port</label>
+				<description>Default is 230 UDP.
+				</description>
+				<default>230</default>
+			</parameter>
+			<parameter name="PREFERRED_LOCAL_PORT_NUMBER" type="integer" required="true">
+				<label>Preferred local port number</label>
+				<description>Default port is 0. It means that it is random.
+				</description>
+				<default>0</default>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="PING_INTERVAL" type="integer" min="1" max="300">
+				<label>Ping interval</label>
+				<description>Interval in seconds to check for device presence.</description>
+				<default>30</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="SUBSCRIBTION_INTERVAL" type="integer" min="1" max="60">
+				<label>Subscription interval</label>
+				<description>Interval in minutes to subcribe Souliss Gateway.
+				</description>
+				<default>2</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="HEALTHY_INTERVAL" type="integer" min="1" max="60">
+				<label>Healthy interval</label>
+				<description>Interval in seconds to send nodes healthy.
+				</description>
+				<default>35</default>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="SEND_INTERVAL" type="integer" min="5" max="5000">
```

Comment:
Should this and some of the other parameters be marked as <required>true</required> ? What happens if someone deletes the  default value does it cause issues?

Reply:
Unnecessary. Cutted away